aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -4,150 +4,124 @@ author: camsoper
-3. Select the **Change Authentication** button.
-    
-    ![Change Authentication Button](./azure-ad-b2c/_static/changeauth.png)
+    :::code language="javascript" source="azure-ad-b2c/sample/appsettings.json" highlight="2-8":::
 
-4. In the **Change Authentication** dialog, select **Individual User Accounts**, and then select **Connect to an existing user store in the cloud** in the dropdown. 
+    - For **Domain**, use the domain of your Azure AD B2C tenant.
+    - For **ClientId**, use the **Application (client) ID** from the app registration you created in your tenant.
+    - Leave all other values as they are.
     
-    ![Change Authentication Dialog](./azure-ad-b2c/_static/changeauthdialog.png)
+1. In *Views/Shared*, create a file named *_LoginPartial.cshtml*. Include the following code:
 
-5. Complete the form with the following values:
-    
-    | Setting                       | Value                                                 |
-    |-------------------------------|-------------------------------------------------------|
-    | **Domain Name**               | *&lt;the domain name of your B2C tenant&gt;*          |
-    | **Application ID**            | *&lt;paste the Application ID from the clipboard&gt;* |
-    | **Callback Path**             | *&lt;use the default value&gt;*                       |
-    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn`                                        |
-    | **Reset password policy**     | `B2C_1_SSPR`                                          |
-    | **Edit profile policy**       | *&lt;leave blank&gt;*                                 |
-    
-    Select the **Copy** link next to **Reply URI** to copy the Reply URI to the clipboard. Select **OK** to close the **Change Authentication** dialog. Select **OK** to create the web app.
+    :::code language="razor" source="azure-ad-b2c/sample/Pages/Shared/_LoginPartial.cshtml":::    
 
-## Finish the B2C app registration
+    In the preceding code:
 
-Return to the browser window with the B2C app properties still open. Change the temporary **Reply URL** specified earlier to the value copied from Visual Studio. Select **Save** at the top of the window.
+    - The partial view checks if the user is authenticated.
+    - Renders a "Sign out" or "Sign in" link as appropriate.
+    - The link points to an action method on the `Account` controller in the `MicrosoftIdentity` area.
 
-> [!TIP]
-> If you didn't copy the Reply URL, use the HTTPS address from the Debug tab in the web project properties, and append the **CallbackPath** value from *appsettings.json*.
+1. In *Views/Shared/_Layout.cshtml*, add the highlighted line within the `<header>` element:
 
-## Configure policies
+    :::code language="razor" source="azure-ad-b2c/sample/Pages/Shared/_Layout.cshtml" range="11-32" highlight="10":::
+ 
+    This renders the *_LoginPartial.cshtml* partial view in every page request that uses this layout.
```

Comment:
```suggestion
    Adding `<partial name="_LoginPartial" />` renders the *_LoginPartial.cshtml* partial view in every page request that uses this layout.
```

Given the default is every page uses this partial, the following is simpler and won't raise the question for newbee's - what pages use the layout. 

```suggestion
    Adding `<partial name="_LoginPartial" />` renders the *_LoginPartial.cshtml* partial view in every page request.
```